news/three-months-at-the-rust-foundation.md

```diff
@@ -0,0 +1,83 @@
+---
+date: 2023-07-01T12:00:00Z
+title: My first three months at the Rust Foundation
+description: Time flies when you're having fun… at the Rust Foundation
+byline: Tobias Bieniek, crates.io team co-lead and Rust Foundation software engineer
+tags:
+  - foundation
+layout: layouts/news.njk
+---
+
+In May 2022, I received an exciting email from the Rust Foundation:
+
+> […] you have been provisionally selected to be one of our first Rust Foundation Fellows
+
+Fast-forward one year, and that fellowship grant has transformed into a full-time role at the Rust Foundation. My primary focus now revolves around the maintenance and feature development for [crates.io](https://crates.io), the official Rust package registry.
```

Comment:
I'm not sure "I've applied because getting money for voluntary open-source work is nice" adds much value 😅

Reply:
I think it would though. Not worded like that but something like this? 

> In April 2022, I received an exciting email from the Rust Foundation:
> […] you have been provisionally selected to be one of our first Rust Foundation Fellows
> 
> 
> As a contributor to the Rust Project for X years, receiving this grant money allowed me to focus more of my time on the open source volunteer work I enjoyed. 
> 
> Fast-forward one year, and that Rust Foundation Fellowship has led to a full-time role at the Rust Foundation where my focus revolves around the maintenance and feature development for [crates.io](https://crates.io/), the official Rust package registry.

Reply:
added, thanks! 👍